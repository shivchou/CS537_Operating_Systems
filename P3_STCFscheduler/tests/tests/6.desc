give_cpu() tie breaking
