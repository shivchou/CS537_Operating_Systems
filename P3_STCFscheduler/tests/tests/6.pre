./edit-makefile.sh ../solution/Makefile test_6 > ../solution/Makefile.test
cp -f tests/test_6.c ../solution/test_6.c
cd ../solution/
make clean
make -f Makefile.test clean
make -f Makefile.test xv6.img
make -f Makefile.test fs.img
cd ../tests
