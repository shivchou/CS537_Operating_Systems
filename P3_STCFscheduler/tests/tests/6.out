XV6_TEST: scheduler : STCF
XV6_TEST: parent start
XV6_TEST: proc 0 start
XV6_TEST: proc 2 start
XV6_TEST: proc 2 end
XV6_TEST: proc 0 end
XV6_TEST: proc 1 start
XV6_TEST: proc 1 end
